.gitignore

```diff
@@ -15,6 +15,8 @@ src/main/resources/docs/
 /*.log.*
 hs_err_pid[0-9]*.log
 
+/config/settings.json
```

Comment:
I think the file path should be .vscode/settings.json?

Reply:
Should just be `.vscode/`